test/status-handler.test.js

```diff
@@ -19,7 +20,53 @@ describe('Status-Handler', function () {
     before(() => {
       const data = helpers.connectDatastar({ mock: true }, mocks.datastar());
       status = new StatusHandler({
-        models: models(data)
+        models: models(data),
+        webhooks: {
+          whatever: [
+            'http://example.com/webhooks'
+          ]
+        }
+      });
+    });
+
+    describe('_getPackageName', function () {
+      it('should return the package name', function () {
+        const pkg = status._getPackageName(fixtures.singleQueued);
+        assume(pkg).equals('whatever');
+      });
+    });
+
+    describe('_shouldSendWebhook', function () {
+      it('should check if a package has registered endpoints', function () {
+        assume(status._shouldSendWebhook('whatever')).equals(true);
+        assume(status._shouldSendWebhook('whatever2')).equals(false);
+      });
+    });
+
+    describe('_isBuildQueued', function () {
+      it('should check if a build is queued', async function () {
+        const findAllStub = sinon.stub(status.models.StatusEvent, 'findAll')
+          .resolves([fixtures.secondEvent, fixtures.singleQueued]);
+        const { pkg, version, env } = fixtures.singleQueued;
+        const result = await status._isBuildQueued({ pkg, version, env });
+        assume(findAllStub).is.calledWith({ pkg, version, env });
+        assume(result).equals(true);
+        sinon.restore();
```

Comment:
Best practice to put `sinon.restore` calls into an `after` or `afterEach` so that you always restore even if the test fails part way through.

Reply:
@decompil3d I know and I could not agree more with you.

However, the reason why I did in this way is to stay consistent with the other pre-existing tests (e.g., see https://github.com/godaddy/warehouse.ai-status-api/blob/master/test/status-handler.test.js#L169 and/or https://github.com/godaddy/warehouse.ai-status-api/blob/master/test/status-handler.test.js#L299)

Aka, I just maintained the existing pattern. I do not want this PR becoming a refactor PR as well. 

Does it make sense?

Reply:
But you're adding new `describe` blocks here, so it's best to just do the right thing instead of introducing even more bad code.

Reply:
Ok yeah, makes sense. Changed this.